doc/source/how-to-run-flower-using-docker.rst

```diff
@@ -126,14 +132,14 @@ Prerequisites
 ~~~~~~~~~~~~~
 
 Before we can start, we need to meet a few prerequisites in our local development environment.
-You can skip the first part if you want to run your client-app instead of the ``app-pytorch``
+You can skip the first part if you want to run your client-app instead of the ``quickstart-pytorch``
```

Comment:
Rename client-app > ClientApp for consistency.

Reply:
Good catch!